src/scss/components/_loading.scss

```diff
@@ -0,0 +1,161 @@
+.bp-loading {
+  margin-left: auto;
+  margin-right: auto;
+  position: relative;
+  width: 40px;
```

Comment:
40px and 25px: in the toolkit we use a variable $m: 0.625rem to easily translate pixel units into REM units. The correct would be to set width and height to 4*$m (40px) and 2.5*$m (25px). Since everything is in REM, all components will scale responsively to the html root's font-size.